content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -4,247 +4,206 @@ reviewers:
 
-The generated `SelfSubjectAccessReview` is:
-```yaml
-apiVersion: authorization.k8s.io/v1
-kind: SelfSubjectAccessReview
-metadata:
-  creationTimestamp: null
-spec:
-  resourceAttributes:
-    group: apps
-    resource: deployments
-    namespace: dev
-    verb: create
-status:
-  allowed: true
-  denied: false
-```
+You must set a command line argument to the API server, to indicate which authorizationmode
+the API server should use.
 
-## Using Flags for Your Authorization Module
+You can use the following modes:
 
-You must include a flag in your policy to indicate which authorization module
-your policies include:
+* `--authorization-mode=ABAC` (Attribute-based access control mode)
+* `--authorization-mode=RBAC` (Role-based access control mode)
+* `--authorization-mode=Node` (Node authorizer)
+* `--authorization-mode=Webhook` (Webhook authorization mode)
+* `--authorization-mode=AlwaysAllow` (always allows requests; carries [security risks](#warning-always-allow))
+* `--authorization-mode=AlwaysDeny` (always denies requests)
 
-The following flags can be used:
+You can choose more than one authorization mode; for example:
+`--authorization-mode=Node,Webhook`
 
-  * `--authorization-mode=ABAC` Attribute-Based Access Control (ABAC) mode allows you to configure policies using local files.
-  * `--authorization-mode=RBAC` Role-based access control (RBAC) mode allows you to create and store policies using the Kubernetes API.
-  * `--authorization-mode=Webhook` WebHook is an HTTP callback mode that allows you to manage authorization using a remote REST endpoint.
-  * `--authorization-mode=Node` Node authorization is a special-purpose authorization mode that specifically authorizes API requests made by kubelets.
-  * `--authorization-mode=AlwaysDeny` This flag blocks all requests. Use this flag only for testing.
-  * `--authorization-mode=AlwaysAllow` This flag allows all requests. Use this flag only if you do not require authorization for your API requests.
+Kubernetes checks authorization modules based on the order that you specify them
+on the API server's command line, so an earlier module has higher priority to allow
+or deny a request.
 
-You can choose more than one authorization module. Modules are checked in order
-so an earlier module has higher priority to allow or deny a request.
+For more information on command line arguments to the API server, read the
+[`kube-apiserver` reference](/docs/reference/command-line-tools-reference/kube-apiserver/).
 
-## Configuring the API Server using an Authorization Config File
+### Configuring the API Server using an authorization config file
 
 {{< feature-state state="alpha" for_k8s_version="v1.29" >}}
 
-The Kubernetes API server's authorizer chain can be configured using a
-configuration file.
+In order to customise the authorizer chain using a configuration file,
+you need to enable the `StructuredAuthorizationConfiguration`
```

Comment:
only need to manually enable for 1.29 and it’s enabled by default in 1.30

Reply:
Until v1.30 is released, these **are** the docs for v1.29

Reply:
v1.30 is released now, and I've rebased